cupyx/tools/install_library.py

```diff
@@ -35,6 +35,7 @@ def _make_cudnn_record(
         cuda_version, public_version, filename_linux, filename_windows):
     major_version = public_version.split('.')[0]
     return {
+        'packaging': 'pip',
```

Comment:
`_wheel.json` is not directly generated from this metadata, so no change is needed in installers.

Reply:
@kmaehashi But I thought you generate the json file by running this script?

Reply:
Currently, the release tool runs the installer with `--action dump` option to extract info from this installer, then generate JSON using that info:
https://github.com/cupy/cupy-release-tools/blob/c9ad0ddb8cc4a5f20d062073c27e77d5748e4deb/dist.py#L404-L413

So installer does not need to have `packaging` field.

Reply:
Ahh ok, didn't realize it's a two-step process. I thought `--action dump` is enough...OK, then I'll remove the changes in the installer, and let you handle `'packaging': 'pip'` in the release tool (which I kinda lost track 😅).

Reply:
Yes, I'm working on it now.  > 'packaging': 'pip' in the release tool

Reply:
Changes in `install_library.py` are reverted.